$DragonFly: src/gnu/usr.bin/cc34/cc_prep/patches/common.opt.patch,v 1.2 2004/06/20 00:10:52 joerg Exp $

diff -u ../gcc.orig/common.opt common.opt
--- ../gcc.orig/common.opt	2004-06-19 22:27:09.000000000 +0200
+++ common.opt	2004-06-20 01:16:04.000000000 +0200
@@ -152,6 +152,10 @@
 Common
 Warn when a variable is unused
 
+Wstack-protector
+Common
+Warn when not issuing stack smashing protection for some reason
+
 aux-info
 Common Separate
 -aux-info <file>	Emit declaration information into <file>
@@ -738,6 +742,14 @@
 Common
 Put zero initialized data in the bss section
 
+fstack-protector
+Common
+Enables stack protection
+
+fstack-protector-all
+Common
+Enables stack protection of every function
+
 g
 Common JoinedOrMissing
 Generate debug information in default format
